libyul/backends/wasm/EVMToEwasmTranslator.cpp

```diff
@@ -68,7 +68,8 @@ Object EVMToEwasmTranslator::run(Object const& _object)
 	Block ast = std::get<Block>(Disambiguator(m_dialect, *_object.analysisInfo)(*_object.code));
 	set<YulString> reservedIdentifiers;
 	NameDispenser nameDispenser{m_dialect, ast, reservedIdentifiers};
-	OptimiserStepContext context{m_dialect, nameDispenser, reservedIdentifiers};
+	// 200 is the default value for optimize-runs
```

Comment:
Please document that it is unused - that is the point here, right?
In addition, can you use `OptimiserSettings::standard().expectedExecutinosPerDeployment`?